nvm.sh

```diff
@@ -1642,88 +1642,86 @@ nvm_get_checksum() {
 }
 
 nvm_print_versions() {
-  local VERSION
-  local LTS
-  local FORMAT
   local NVM_CURRENT
-  local NVM_LATEST_LTS_COLOR
-  local NVM_OLD_LTS_COLOR
-
   local INSTALLED_COLOR
   local SYSTEM_COLOR
   local CURRENT_COLOR
   local NOT_INSTALLED_COLOR
   local DEFAULT_COLOR
-  local LTS_COLOR
+  local NVM_HAS_COLORS
 
+  NVM_CURRENT=$(nvm_ls_current)
   INSTALLED_COLOR=$(nvm_get_colors 1)
   SYSTEM_COLOR=$(nvm_get_colors 2)
   CURRENT_COLOR=$(nvm_get_colors 3)
   NOT_INSTALLED_COLOR=$(nvm_get_colors 4)
   DEFAULT_COLOR=$(nvm_get_colors 5)
-  LTS_COLOR=$(nvm_get_colors 6)
```

Comment:
this seems like it removes support for overriding the LTS color?

Reply:
Actually the variable `LTS_COLOR` was not used in `mvn_print_versions`.

In total `LTS_COLOR` appeared at 7 places, 4 in `nvm_print_formatted_alias` (irrelevant to this pr), 1 in the main `nvm` function, and 2 in this `mvn_print_versions`, but that was only the declaration and initialization.

Meanwhile in `nvm_print_versions`, it used separate `NVM_LATEST_LTS_COLOR` and `NVM_OLD_LTS_COLOR` for coloring the lts versions, with [the latter set to `$DEFAULT_COLOR` (white) as the initial value here][1], which [had been like that][2] when pr #2291 was merged.

[1]: https://github.com/nvm-sh/nvm/blob/a82edf40ba72a15b1d40822735942141547f38e2/nvm.sh#L1668
[2]: https://github.com/nvm-sh/nvm/blame/a82edf40ba72a15b1d40822735942141547f38e2/nvm.sh#L1668

Currently this pr focuses on maintaining existing behavior. where the old lts names are displayed with default color. But I do see that in `nvm ls`, more specifically `nvm_print_formatted_alias` it uses `LTS_COLOR` as mentioned above. So the current behavior is likely unintended, which surely I can fix.

Reply:
oh interesting, thanks - good catch.